README.md

```diff
@@ -347,4 +392,18 @@ class MyCustomApplicationLoader extends ApplicationLoader {
     (new GuiceApplicationLoader).load(newContext)
   }
 }
-```
\ No newline at end of file
+```
+
+#### Play 2.3
+
+```scala
+import play.api._
+import com.typesafe.conductr.bundlelib.play.Env
+
+object Global extends GlobalSettings {
+  val totalConfiguration = super.configuration ++ Configuration(Env.asConfig)
```

Comment:
Actually, I'm going to leave this alone. I'm not that fussed about 2.3 and I know what we have will work (I'm sure yours works too, but I can't handle more workload debugging old things that worked presently!).